packages/orchestration/src/examples/swap-anything.flows.js

```
@@ -207,16 +233,22 @@ export const swapAnythingViaHook = async (
   trace(`got info for chain: osmosis ${osmosisChainInfo}`);
   trace(osmosisChainInfo);
 
-  const memo = buildXCSMemo(memoArgs);
+  try {
+    const memo = buildXCSMemo(memoArgs);
 
-  await sharedLocalAccount.transfer(
-    {
-      value: destAddr,
-      encoding: 'bech32',
-      chainId: /** @type {CosmosChainInfo} */ (osmosisChainInfo).chainId,
-    },
-    { denom, value },
-    { memo },
-  );
+    await sharedLocalAccount.transfer(
+      {
+        value: destAddr,
+        encoding: 'bech32',
+        chainId: /** @type {CosmosChainInfo} */ (osmosisChainInfo).chainId,
+      },
+      { denom, value },
+      { memo },
+    );
+    trace(`completed transfer to ${destAddr}`);
+  } catch (e) {
+    void log(`transfer error ${q(e)}`, 'error');
+    return recoverFailedTransfer(e);
+  }
```

Should re-throw, yes?

NEEDSTEST

---

Could you please clarify if I understood your comment correctly?

Currently, if an error is catch on the try catch block, it will log the error and calls recoverFailedTransfer, which itself throws an error after attempting recovery.
Are you suggesting that the error should be explicitly thrown an error after recoverFailedTransfer, to ensure the error is not silently swallowed if the transfer to the tokens owner fails?